mmv1/products/gkebackup/BackupPlan.yaml

```diff
@@ -260,3 +260,13 @@ properties:
     output: true
     description: |
       The number of Kubernetes Pods backed up in the last successful Backup created via this BackupPlan.
+  - !ruby/object:Api::Type::String
+    name: state
```

Comment:
Do we think these fields will be useful to Terraform users? Would you mind sharing the API documentation of these fields or what are the possible values for `state`? as I imagine we would always get `state: READY` for successful resource creation/update, otherwise we would end up with errors and might not write these fields to terraform state.

Reply:
Public facing documentation isn't complete for api change. This is the description of the fields

STATE_UNSPECIFIED: Default first value for Enums.
CLUSTER_PENDING: Waiting for cluster state to be RUNNING.
PROVISIONING: The BackupPlan is in the process of being created.
READY: BackupPlan is ready to schedule backups
FAILED: BackupPlan creation has failed.
DEACTIVATED: The BackupPlan has been deactivated.
DELETING: The BackupPlan is in the process of being deleted.

Note that CLUSTER_PENDING state can potentially take a significant amount of time (30-40 min) depending on how the agent is being installed. State Reason during FAILED state can be helpful for customers to understand where the failure occurred (especially since the operation can take 30-40 min), and how to fix it

Reply:
@shuyama1 let me know if you think this isn't a useful field for terraform users